tpm2/encoding_test.go

```diff
@@ -142,9 +141,8 @@ func TestEncodeCreate(t *testing.T) {
 				KeyBits: 128,
 				Mode:    AlgCFB,
 			},
-			KeyBits:  1024,
-			Exponent: uint32(0x00010001),
-			Modulus:  big.NewInt(0),
+			KeyBits:     1024,
+			ExponentRaw: 0x00010001,
```

Comment:
maybe whack this badboi in a constant (is this `defaultRSAExponent`)?

Reply:
Done, and also fixed the `0x00030072` raw value.